src/frontend/src/icp/utils/icrc.utils.ts

```diff
@@ -131,3 +131,6 @@ export const buildIcrcCustomTokenMetadataPseudoResponse = ({
 		...(nonNullish(icon) ? [icon] : [])
 	];
 };
+
+export const isIcrcCustomToken = (token: Partial<IcrcCustomToken>): token is IcrcCustomToken =>
```

Comment:
A token can be of type Icrc Custom if disabled too, so maybe either rename the util or revisit its logic.

Reply:
yes, agreed, but that function just checks the existence of the properties (not their value) and that would identify the type

```typescript
export const isIcrcCustomToken = (token: Partial<IcrcCustomToken>): token is IcrcCustomToken =>
	nonNullish(token.ledgerCanisterId) && nonNullish(token.enabled);
```

shall i call it 

isTypeIcrcCustomToken

Reply:
>  but that function just checks the existence

If the goals is to check the existence of the properties we should not test if they are undefined null or defined but rather check if they are present in my opinion.

For example:

```
ledgerCanisterId in token && indexCanisterId in token && enabled in token
```